### ISSUE TITLE
[IMPROVEMENT] Nullable icons should have a better contrast

### ISSUE THREAD

**Issue by nocodelab** [TiagoSilvaPereira/vemto-issues#199]
Nullable icons should have a better contrast ratio in order to be readable correctly.

<img width="311" alt="Screenshot 2021-07-13 at 20 16 21" src="https://user-images.githubusercontent.com/4540118/125504376-8c9f71a5-6b5b-4f37-95c3-f1ba95ef91ad.png">

Please find below a WCAG compliance test:
<img width="512" alt="Screenshot 2021-07-13 at 20 05 53" src="https://user-images.githubusercontent.com/4540118/125503001-19f13228-f692-445a-b699-a8fd78b90454.png">

A solution would be to use the same colo scheme of the tables diagram:
<img width="304" alt="Screenshot 2021-07-13 at 20 17 56" src="https://user-images.githubusercontent.com/4540118/125504443-30f02c0a-372c-44f9-a0a5-51b971f57f81.png">

Cheers!

**Comment by TiagoSilvaPereira** [comment]
Hi @nocodelab thank you very much for your suggestion

**Comment by TiagoSilvaPereira** [comment]
@nocodelab fixed on 0.9.10
